composer.json

```diff
@@ -37,12 +38,12 @@
     "drupal/scheduler": "*",
     "drupal/vuejs": "^1.0@beta",
     "drupal/workspace": "^1.0@beta",
-    "drupal/uuid_url": "^1.0"
+    "drupal/uuid_url": "^1.0",
+    "drupal/coder": "8.3.8"
```

Comment:
should be in require-dev - it's only for running tests but projects using this can use whatever code version.